src/components/TodoList/TodoList.tsx

```diff
@@ -0,0 +1,49 @@
+import { FC, memo } from 'react';
+import { Todo } from '../../types/Todo';
+import { TodoItem } from '../TodoItem/TodoItem';
+
+export interface TodoListProps {
+  todos: Todo[],
+  onDeleteTodo: (todoId: number) => void;
+  tempTodo: Todo | null,
+  onDeletingTodoIds: number[],
```

Comment:
rename this property. for now it looks like a function
```suggestion
  deletingTodoIds: number[],
```